lib/src/http/checkToken.ts

```diff
@@ -0,0 +1,65 @@
+ { defaultInterceptor } from "./client";
```

Comment:
Looks like this suggestion got mangled.
```suggestion
import { defaultInterceptor } from "./client";
```